vmdb/app/controllers/catalog_controller.rb

```diff
@@ -665,6 +669,12 @@ def process_sts(sts, task, display_name = nil)
 
   private      #######################
 
+  def class_service_template(prov_type)
+    prov_type.starts_with?('generic') ?
```

Comment:
Why do we need to check `prov_type` first? This should not be called if it is not `generic` in the first place. If `prove_type` does not start with `generic` it still returns `ServiceTemplate`, which probably is not what we want.